src/dbnode/storage/index/options.go

```diff
@@ -98,11 +98,13 @@ type opts struct {
 	docArrayPool                    doc.DocumentArrayPool
 	foregroundCompactionPlannerOpts compaction.PlannerOptions
 	backgroundCompactionPlannerOpts compaction.PlannerOptions
+	postingsListCache               *PostingsListCache
```

Comment:
could you add a `!= nil` in the `Validate()`

Reply:
sure